src/CMakeLists.txt

```diff
@@ -915,7 +918,7 @@ else()
 		if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 			set(OTHER_FLAGS "${OTHER_FLAGS} -mthreads")
 		endif()
-		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_WIN32_WINNT=0x0601 -DWIN32_LEAN_AND_MEAN")
+		add_compile_definitions(_WIN32_WINNT=0x0601 WIN32_LEAN_AND_MEAN)
```

Comment:
It looks like `_WIN32_WINNT=0x0601` is set twice, but they are for different cases. This one is for MinGW, and the earlier one is for MSVC.